docs/superpowers/plans/2026-05-11-sync-agnostic.md

```
@@ -0,0 +1,536 @@
+---
+
+### Task 7: Update README
+
+**Files:**
+- Modify: `README.md`
+
+- [ ] **Step 1: Update the CLI section**
+
+Remove `wd sync status`, `wd sync now`, `wd sync doctor`, and `wd sync resolve` from the CLI code block. The updated block should be:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)